example/burrow_integration/integration_eaxample.py

```diff
@@ -0,0 +1,171 @@
+import os
+import binascii
+from iroha import IrohaCrypto
+from iroha import Iroha, IrohaGrpc
+from iroha.primitive_pb2 import can_set_my_account_detail
+import sys
+from Crypto.Hash import keccak
+
+if sys.version_info[0] < 3:
+    raise Exception('Python 3 or a more recent version is required.')
+
+# Here is the information about the environment and admin account information:
+IROHA_HOST_ADDR = os.getenv('IROHA_HOST_ADDR', '127.0.0.1')
+IROHA_PORT = os.getenv('IROHA_PORT', '50051')
+ADMIN_ACCOUNT_ID = os.getenv('ADMIN_ACCOUNT_ID', 'admin@test')
+ADMIN_PRIVATE_KEY = os.getenv(
+    'ADMIN_PRIVATE_KEY', 'f101537e319568c765b2cc89698325604991dca57b9716b58016b253506cab70')
+
+iroha = Iroha(ADMIN_ACCOUNT_ID)
+net = IrohaGrpc('{}:{}'.format(IROHA_HOST_ADDR, IROHA_PORT))
+
+test_private_key= '7e00405ece477bb6dd9b03a78eee4e708afc2f5bcdce399573a5958942f4a390'
+
+def trace(func):
+    """
+    A decorator for tracing methods' begin/end execution points
+    """
+
+    def tracer(*args, **kwargs):
+        name = func.__name__
+        print('\tEntering "{}"'.format(name))
+        result = func(*args, **kwargs)
+        print('\tLeaving "{}"'.format(name))
+        return result
+
+    return tracer
+
+def make_number_hex_left_padded(number: str, width: int=64):
+    number_hex = '{:x}'.format(number)
+    return str(number_hex).zfill(width)
+
+
+def left_padded_address_of_param(param_index: int, number_of_params: int, width: int=64 ):
+    '''Specifies the position of each argument according to Contract AbI specifications.'''
+    bits_offset = 32 * number_of_params
+    bits_per_param = 64
+    bits_for_the_param = bits_offset + bits_per_param * param_index
+    return make_number_hex_left_padded(bits_for_the_param, width)
+
+
+def argument_encoding(arg) :
+    '''Encodes the argument according to Contract ABI specifications.'''
+    encoded_argument = str(hex(len(arg)))[2:].zfill(64)
+    encoded_argument = encoded_argument+arg.encode('utf8').hex().ljust(64,'0').upper()
+    return encoded_argument
+
+def get_first_four_bytes_of_keccak(function) :
```

Comment:
the name of param can be clearly understandable, I suggest to change into `function_signature: str`